style.css

```diff
@@ -0,0 +1,53 @@
+#something {
+  text-align: center;
+}
+
+#somethinmore {
+  align-items: center;
+  padding-bottom: 2em;
+  padding-top: 2em;
+}
+
+body {
+  background-color: rgb(230, 217, 201);
+}
+
+#form {
+  padding-top: 7em;
+  padding-bottom: 2em;
+  align-items: center;
+  text-align: center;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  display: flex;
+  top: 0;
+  left: 0;
+  min-height: 30px;
+  justify-content: space-around;
+  background-color: rgb(230, 217, 201);
+  padding: 8px 2px;
+}
+
+@media (max-width: 600px) {
+  header {
+    flex-wrap: wrap;
+  }
+}
+
+#evenmore {
+  border: 3px;
+  border-radius: 2px;
+  text-align: center;
+}
```

Comment:
Missed Flex

Reply:
**_Missed Flex_**

- [ ] I think the comment could be more professionally explained. "Missed Flex" could mean anything, rather I suggest saying, **_"I think you can use CSS Flex to display the items"_** and at least provide a link on how to use CSS Flex.